spyder/utils/debug.py

```diff
@@ -33,7 +33,7 @@ def log_last_error(fname, context=None):
         print("Context", file=fd)
         print("-------", file=fd)
         print("", file=fd)
-        print(context, file=fd)
+        print(u' '.join(context).encode('utf-8').strip(), file=fd)

```

Comment:
This was needed because an error was thrown in the python console when changing the directory to one with non - ascii name. Actually was really strange when I saw it. I also saw that the error causes that a process in the python console crashes and then, if you try to change again the working directory, Spyder crashes too.